### PR TITLE
[JA] Privacy Sandbox Translations

### PR DESCRIPTION
**Pull request by ilyaspiridonov**
Translations by Alconost (Post-Edited Machine Translation) 
[See review request for index.md on GitLocalize](https://gitlocalize.com/repo/7832/ja/review/56398)
[See review request for index.md on GitLocalize](https://gitlocalize.com/repo/7832/ja/review/56400)
[See review request for index.md on GitLocalize](https://gitlocalize.com/repo/7832/ja/review/56401)
[See review request for index.md on GitLocalize](https://gitlocalize.com/repo/7832/ja/review/56402)
[See review request for index.md on GitLocalize](https://gitlocalize.com/repo/7832/ja/review/56403)
[See review request for index.md on GitLocalize](https://gitlocalize.com/repo/7832/ja/review/56404)
[See review request for index.md on GitLocalize](https://gitlocalize.com/repo/7832/ja/review/56405)
[See review request for index.md on GitLocalize](https://gitlocalize.com/repo/7832/ja/review/56407)

### CODE REVIEW

**Comment by ilyaspiridonov** [comment]
@agektmr

File: site/ja/docs/privacy-sandbox/trust-tokens/index.md [hunk 1]
```diff
@@ -1,95 +1,90 @@
 ---
-layout: 'layouts/doc-post.njk'
-title: 'トラストトークン'
-subhead: >
+layout: layouts/doc-post.njk
+title: プライベートステートトークン
+subhead: |2
```
Comment (agektmr) [inline site/ja/docs/privacy-sandbox/trust-tokens/index.md:4]:
```suggestion
subhead: >
```

File: site/ja/docs/privacy-sandbox/trust-tokens/index.md [hunk 2]
```diff
@@ -1,95 +1,90 @@
...
+
   限定的な情報をブラウジングコンテキストから別のブラウジングコンテキストに（サイト間など）伝達できるようにすることで、パッシブ追跡を行わずに不正行為に対抗できるようにするための API。
-description: >
+description: |2
```
Comment (agektmr) [inline site/ja/docs/privacy-sandbox/trust-tokens/index.md:7]:
```suggestion
description: >
```

File: site/ja/docs/privacy-sandbox/topics/index.md [hunk 3]
```diff
@@ -1,12 +1,14 @@
 ---
-layout: 'layouts/doc-post.njk'
-title: 'Topics API'
-subhead: >
-  ユーザーがアクセスしたサイトを追跡せずに、インタレストベース広告を可能にします。
-description: >
-  ユーザーがアクセスしたサイトを追跡せずに、インタレストベース広告を可能にするメカニズムの提案。
+layout: layouts/doc-post.njk
+title: 'Topics API: 開発者ガイド'
+subhead: |2-
```
Comment (agektmr) [inline site/ja/docs/privacy-sandbox/topics/index.md:4]:
```suggestion
subhead: >
```

File: site/ja/docs/privacy-sandbox/topics/index.md [hunk 4]
```diff
@@ -1,12 +1,14 @@
...
+
+  Topics のデモを試して、API と、フラグを使用して Topics を実行する方法、またはオリジントライアルに参加する方法について学びます。
+description: |2-
```
Comment (agektmr) [inline site/ja/docs/privacy-sandbox/topics/index.md:7]:
```suggestion
description: >
```

File: site/ja/docs/privacy-sandbox/shared-storage/index.md [hunk 5]
```diff
@@ -1,177 +1,151 @@
 ---
-layout: 'layouts/doc-post.njk'
-title: '共有ストレージ'
-subhead: >
+layout: layouts/doc-post.njk
+title: 共有ストレージ
+subhead: |2-
```
Comment (agektmr) [inline site/ja/docs/privacy-sandbox/shared-storage/index.md:4]:
```suggestion
subhead: >
```

File: site/ja/docs/privacy-sandbox/shared-storage/index.md [hunk 6]
```diff
@@ -1,177 +1,151 @@
...
+
   安全な環境で、パーティション化されていないクロスサイトデータへのアクセスを許可します。
-description: >
+description: |2-
```
Comment (agektmr) [inline site/ja/docs/privacy-sandbox/shared-storage/index.md:7]:
```suggestion
description: >
```

**Review by agektmr — CHANGES_REQUESTED** [review]
Could you please fix these known issues before you submit a pull request?

Thank you!

**Review by agektmr — APPROVED** [review]
LGTM

**Review by anusmitaray — APPROVED** [review]

**Pull request merged**